src/sidebar/components/group-list-section.js

```diff
@@ -1,23 +1,69 @@
 'use strict';
 
-// @ngInject
-function GroupListSectionController() {
-  this.isSelectable = function(groupId) {
-    const group = this.sectionGroups.find(g => g.id === groupId);
-    return !this.disableOosGroupSelection || group.isScopedToUri;
-  };
+const { Fragment, createElement } = require('preact');
+const propTypes = require('prop-types');
+
+const GroupListItem = require('./group-list-item');
+const GroupListItemOutOfScope = require('./group-list-item-out-of-scope');
+
+/**
+ * A labeled section of the groups list.
+ */
+function GroupListSection({
+  analytics,
+  disableOosGroupSelection,
+  heading,
+  sectionGroups,
```

Comment:
It is a namespace reason. Because groups is already used by the parent controller, it is inherited on scope and made available in the child controller so we must use a different name.

Reply:
> . Because groups is already used by the parent controller, it is inherited on scope 

That is not the case when using AngularJS _components_ as we do, since they have "isolated scopes". It _used_ to be the case with "old style" AngularJS (circa 2012-2015) though, so you will still see many articles on the web which mention it and I don't blame you for getting confused by that!

Reply:
I've changed the name to just `groups` now.